.travis.yml

```diff
@@ -24,9 +24,4 @@ before_script:
   - sh -e /etc/init.d/xvfb start
   - sleep 3
 script:
-  - npm run lint
-  - npm test
-  - npm run build:prod
-  - npm run server:prod:ci &
-  - sleep 3
-  - npm run e2e
+  - npm run ci
```

Comment:
I would also suggest running the two build / test steps individually. 

1.) Most of us run multiple test containers, I always run my jit / aot validations in parallel.
2.) If is step is going to fail, it's probably going to be AoT, splitting them just means less log output to go dig through to find the failure.